book/embeds.md

```diff
@@ -834,7 +834,7 @@ class Tab:
         if self.needs_accessibility:
             # ...
 
-        if self.pending_hover:
+        if self.needs_paint:
```

Comment:
Why this change?

Reply:
Yeah, this is a good question. As far as I can tell, this code block was only passing before by accident; `pending_hover` doesn't actually live here.

Reply:
Oh, yeah, right, and it stopped passing because I deleted a bunch of unchanged methods and so it stopped passing by accidentally picking up part of them.

Reply:
Was one of them actually not changed?

Reply:
Ok, so here's the full story. In Chapter 14, we add handling for `pending_hover`. That code lives in `Browser`. So there's code somewhere in `Browser` (specifically, its `paint_draw_list` method) with the code `if self.pending_hover`. That code works, and it works in the same way in Chapter 15.

At some point in the past, things were different, and `pending_hover` was handled in `Tab`. I have no idea how far back this was—it may even be pre-publication of that chapter. Anyway, at that point, the line `if self.pending_hover` made it into that block in `embeds.md`. Later the pending hover code was moved to `Browser`, but the block in `embeds.md` was never updated and it accidentally kept passing, because it was _somewhere_ in the file, namely inside `Browser.paint_draw_list`.

In this PR, I removed `paint_draw_list` because the patch mechanism is now used. That made the block fail. I fixed it up to match the current actual contents of that method.

In short, this change fixes a bug that was already in `main` but got in the way of actually pushing through this PR.